st2tests/integration/mistral/test_filters.py

```diff
@@ -0,0 +1,373 @@
+            parameters={
+                "input_str": REGEX_SEARCH_STR_2,
+                "replacement_str": "foo"
+            }
+        )
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertEqual(
+            execution.result['result_jinja'],
+            "foo is your address. My address is foo."
+        )
+        self.assertEqual(execution.result['result_yaql'], "foo is your address. My address is foo.")
+
+
+class RegexSearchFiltersTest(base.TestWorkflowExecution):
+
+    def test_regex_search(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_search',
+                                           parameters={"input_str": REGEX_SEARCH_STR})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertTrue(execution.result['result_jinja'])
+        self.assertTrue(execution.result['result_yaql'])
+
+    def test_regex_nosearch(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_search',
+                                           parameters={"input_str": REGEX_SEARCH_STR_3})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertFalse(execution.result['result_jinja'])
+        self.assertFalse(execution.result['result_yaql'])
+
+
+class RegexSubstringFiltersTest(base.TestWorkflowExecution):
+
+    def test_regex_substring(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_substring',
+                                           parameters={"input_str": REGEX_SEARCH_STR})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertEqual(execution.result['result_jinja'], '567 Elsewhere Dr')
+        self.assertEqual(execution.result['result_yaql'], '567 Elsewhere Dr')
+        self.assertEqual(execution.result['result_jinja_index_1'], '123 Somewhere Ave')
+        self.assertEqual(execution.result['result_yaql_index_1'], '123 Somewhere Ave')
+
+
+class ToHumanTimeFromSecondsFiltersTest(base.TestWorkflowExecution):
+
+    def test_to_human_time_from_seconds(self):
+        execution = self._execute_workflow(
+            'examples.mistral-customfilters-to_human_time_from_seconds',
+            parameters={"seconds": 4587}
+        )
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertEqual(execution.result['result_jinja'], '1h16m27s')
+        self.assertEqual(execution.result['result_yaql'], '1h16m27s')
+
+
+class UseNoneFiltersTest(base.TestWorkflowExecution):
+
+    def test_use_none(self):
+        inputs = {'input_str': 'foo'}
+        execution = self._execute_workflow(
+            'examples.mistral-customfilters-use_none', parameters=inputs
+        )
```

Comment:
Ah - good catch, I meant to circle back and do this. My first thought was to literally pass `None` as a parameter but there's no JSON schema type to support it, so I went with this approach so I could just create a new variable in the workflow that's `null`. I realized after this that an empty string would suffice just as well (duh), just forgot to go back and simplify. Will do that next.